man/adoc/bpftrace.adoc

```diff
@@ -3791,6 +3791,12 @@ Available options:
 If the DebugInfo was rewritten by a post-linkage optimisation tool (like BOLT or AutoFDO), it might yield an incorrect address for a probe location.
 This config can force using the Symbol Table, for when the DebugInfo returns invalid addresses.
 
+=== use_blazesym
+
+Default: `0` if built without blazesym support, `1` otherwise
```

Comment:
Hmm, I wonder if we shouldn't just decide to build with blazesym by default and then this can default to 1.

Reply:
Not sure we should make it the default right away. Why not do it as a second step?

Reply:
Sure, we can wait. No big deal.